src/Voron/Data/Fixed/FixedSizeTree.cs

```diff
@@ -51,6 +51,9 @@ public static ushort GetValueSize(LowLevelTransaction tx, Tree parent, Slice tre
 
         public FixedSizeTree(LowLevelTransaction tx, Tree parent, Slice treeName, ushort valSize)
         {
+            if (treeName.Array == null)
```

Comment:
I don't really like this, but we'll keep it for now. We need to look into this, since this is probably going to do a lot of allocations in practice.